src/cirrus/plugins/publishers/jenkins.py

```diff
@@ -36,10 +37,11 @@ def publish(self, opts, doc_artifact):
         doc_job = default
         doc_var = archive
         arc_var = ARCNAME
-        extra_vars = [
-            {"name": varname, "value": varvalue},
-            {"name": varname1, "value": varvalue1}
-        ]
+        extra_vars = True
+
+        [extra_vars]
```

Comment:
This is an ambigious name, maybe make it something like jenkins_docs_extra_vars ? 
Aim to make the config as obvious and readable as possible.